### ISSUE TITLE
Fries-like Android Style Transition

### ISSUE THREAD

**Issue by diorahman** [techjacker/Backbone-Responsive-CSS3-Page-Transitions#3]
Hi,

Is it possible to modify the transition style? e.g. I want to have a nice Android page transition just like in fries http://jaunesarmiento.me/fries/

Thanks

diorahman

**Comment by techjacker** [comment]
Yes, you should be able to change the animation by just editing/overriding the stylesheet.

In fact it should a lot easier as it's just a fade so you could keep the page width at 100% and just position the new page behind the old one.
